### PR TITLE
Correct / improve SymfonyProfilerController

### PR DESCRIPTION
**Pull request by odolbeau**

### CODE REVIEW

File: phpstan-baseline.neon [hunk 1]
```diff
@@ -149,3 +149,8 @@ parameters:
             message: "#^Call to an undefined method Symfony\\\\Component\\\\Translation\\\\TranslatorBagInterface\\|Symfony\\\\Contracts\\\\Translation\\\\TranslatorInterface\\:\\:transChoice\\(\\)\\.$#"
             count: 2
             path: Twig/TranslationExtension.php
+
+        -
+            message: "#^Call to method getValue\\(\\) on an unknown class Symfony\\\\Component\\\\Translation\\\\DataCollector\\\\Data\\.$#"
```
Comment (odolbeau) [inline phpstan-baseline.neon]: Will become useless once missing use will be added in symfony
See: https://github.com/symfony/symfony/pull/35122

Comment (rvanlaak) [reply phpstan-baseline.neon]: PHPStan will also give an error when a message is not used anymore 👍 Great way of gradually improving code quality  

compliments to @ondrejmirtes 👏

File: Controller/SymfonyProfilerController.php [hunk 2]
```diff
@@ -122,30 +123,25 @@ public function createAssetsAction(Request $request, string $token): Response
...
         }
         $message = SfProfilerMessage::create($collectorMessages[$messageId]);
 
         if (DataCollectorTranslator::MESSAGE_EQUALS_FALLBACK === $message->getState()) {
-            $message->setLocale($profile->getCollector('request')->getLocale())
-                ->setTranslation(\sprintf('[%s]', $message->getTranslation()));
+            /** @var \Symfony\Component\HttpKernel\DataCollector\RequestDataCollector */
+            $requestCollector = $this->profiler->loadProfile($token)->getCollector('request');
+
+            $message
```
Comment (rvanlaak) [inline Controller/SymfonyProfilerController.php]: What about reducing fluent usage? That way you don't need to rely on the message returning itself.

```
$message->setLocale()
$message->setTranslation()
```

File: Resources/config/symfony_profiler.yaml [hunk 3]
```diff
@@ -6,6 +6,7 @@ services:
       - { name: 'data_collector', template: "@Translation/SymfonyProfiler/translation.html.twig", id: "translation", priority: 200 }
 
   Translation\Bundle\Controller\SymfonyProfilerController:
+    autowire: true
```
Comment (rvanlaak) [inline Resources/config/symfony_profiler.yaml]: What about moving this to a `_defaults` section?

```
_defaults:
    autowire: true
    autoconfigure: true
```

File: phpstan-baseline.neon (hunk 1 above)
Comment (ondrejmirtes) [reply phpstan-baseline.neon]: Thank you 😊

File: Controller/SymfonyProfilerController.php (hunk 2 above)
Comment (odolbeau) [reply Controller/SymfonyProfilerController.php]: I kept the original syntax.

**Pull request merged**